node/buffer.ts

```diff
@@ -273,6 +273,20 @@ export class Buffer extends Uint8Array {
     this.boundsError(byteLength, 4, "byteLength");
   }
 
+  /**
+   * Reads byteLength number of bytes from buf at the specified offset and interprets
+   * the result as an unsigned, little-endian integer supporting up to 32 bits of accuracy.
```

Comment:
I think doc should say `supporting up to 48 bits of accuracy` ref: https://nodejs.org/api/buffer.html#buffer_buf_readuintle_offset_bytelength

The purpose of std/node is the compatibility to node.js, not fixing it or providing our own design.

Reply:
Yes, i will fixe that.